_data/projects/Docker-Kubernetes.yml

```diff
@@ -12,6 +11,3 @@ tags:
 upforgrabs:
   name: Up-for-Grabs
   link: https://github.com/NishkarshRaj/Docker-Kubernetes/labels/Up-for-Grabs
-stats:
```

Comment:
This section is fine to keep around - can you elaborate on why it's being deleted here?

Reply:
Actually, I was 3 commits ahead and 196 commits behind behind on my fork of the upstream and even after fetching and merging the upstream, I was still 3 commits ahead because it was having conflicts.
I thought the stats will be automatically added after the next pull request is merged so went on to remove it on manual merge.

Reply:
> I was still 3 commits ahead because it was having conflicts.

This is likely because we squash each PR into a new commit, which might lead to some confusion from Git because you're also working in the default branch. I can rebase this PR branch if you'd like, and for future PRs I'd work from a new branch each time (which is easier to cleanup).

> I thought the stats will be automatically added after the next pull request is merged so went on to remove it on manual merge.

This is true, but it would help keep PR reviews simple by focusing on what needs to be updated.

Reply:
> I can rebase this PR branch if you'd like, and for future PRs I'd work from a new branch each time (which is easier to cleanup).

Sure, please help me out on this PR with a rebase and I'll surely work on new branches for further PRs.

Reply:
> This is true, but it would help keep PR reviews simple by focusing on what needs to be updated.

I understand the diff must be relevant to the PR. I will keep this point in mind next time :+1:

Reply:
> Sure, please help me out on this PR with a rebase and I'll surely work on new branches for further PRs.

I've force-pushed to your `gh-pages` branch to drop the unimportant commits.

After this is merged, I'd recommend resetting to the upstream `gh-pages` to get the latest changes:

```
$ git checkout gh-pages
$ git reset upstream/gh-pages --force
```

Reply:
@shiftkey I have synchronized my fork with latest upstream state.
Thanks alot for the help 🎉